astro/src/pages/accessibility-statement.astro

```diff
@@ -0,0 +1,26 @@
+---
+import ExternalLink from '../components/ExternalLink.astro';
+import Layout from '../layouts/Layout.astro';
+import ThemedSection from '../components/ThemedSection.astro';
+---
+
+<Layout title="Accessibility Statement">
+ <ThemedSection style="secondary">
+  <h2>Accessibility And Accommodations</h2>
+  <p>At Accessible Community, we believe that our commitment to creating accessible communities starts with us so we strive to provide accessible content and services.</p>
+  <p>We test our websites and services against
+   <ExternalLink
+    href="https://www.w3.org/TR/WCAG22/"
+    target="blank"
+    >WCAG 2.2 A, AA, and AAA criteria</ExternalLink>
+   and
+   <ExternalLink
```

Comment:
For links with a lot of text, please add the `inline={true}` attribute to the `ExternalLink` component, as it allows the text to wrap.

Reply:
I was wondering about that. Looks much better!